docs/remote_svs.rst

```diff
@@ -0,0 +1,189 @@
+Accessing SecureDrop Remotely
+=============================
+
+While it's necessary for SecureDrop servers to be hosted on-premise within your
+organization, and for administrators to retain direct physical access in order
+to perform any necessary maintenance or troubleshooting tasks, it is possible
+for journalists to use SecureDrop in remote locations.
```

Comment:
Since the chapter is broadly about remote access and not strictly scoped to the SVS, I would suggest adding a paragraph about SSH access, clarifying that in the default configuration, app/mon servers can be accessed remotely from anywhere via the Admin Workstation.  We can also cross-link the SSH-over-LAN docs in case people need to change the settings (that's referenced way down in the doc, but I think we can state this upfront).